packages/core/admin/admin/src/components/AutoReloadOverlayBlocker/index.js

```diff
@@ -1,22 +1,14 @@
-import React, { useEffect, useState } from 'react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import ReactDOM from 'react-dom';
-import Content from './Content';
-import Overlay from './Overlay';
-import Wrapper from './Wrapper';
-
-const overlayContainer = document.createElement('div');
-const ID = 'autoReloadOverlayBlocker';
-overlayContainer.setAttribute('id', ID);
-
-const AutoReloadOverlayBlocker = () => {
+import React, { useEffect, useRef, useState } from 'react';
+import { AutoReloadOverlayBockerContext } from '@strapi/helper-plugin';
+import PropTypes from 'prop-types';
+import Portal from './Portal';
+
+const AutoReloadOverlayBlocker = ({ children }) => {
```

Comment:
Can we call component providing a context xxxProvider?

Reply:
Sure!